src/main/java/teammates/ui/webapi/action/SearchInstructorsAction.java

```diff
@@ -26,13 +29,34 @@ public void checkSpecificAccessControl() {
         }
     }
 
+    private void addAdditionalSearchFields(InstructorsData instructorsData, List<InstructorAttributes> instructors) {
+        instructorsData.getInstructors()
+                .forEach((InstructorData data) -> {
```

Comment:
```suggestion
                .forEach((InstructorData instructor) -> {
```

`data` is too general.

Reply:
fixed